apps/io-wallet-web-func/src/infra/http/handlers/get-current-wallet-instance-status.ts

```diff
@@ -0,0 +1,45 @@
+import { getCurrentWalletInstance } from "@/wallet-instance";
+import * as H from "@pagopa/handler-kit";
+import { lookup } from "fp-ts/Record";
+import * as E from "fp-ts/lib/Either";
+import * as RTE from "fp-ts/lib/ReaderTaskEither";
+import { flow, pipe } from "fp-ts/lib/function";
+import { logErrorAndReturnResponse } from "io-wallet-common/http-response";
+import { User } from "io-wallet-common/user";
+
+// TODO: SIW-1266. Now in the `authorization` header there is the pdv tokenizer id, later there will be the authentication token
+// Move the code to a more central part
```

Comment:
What do you think about throwing an error instead of temporarily using an id?

Reply:
why? This way we wouldn't be able to test the function